examples/mxnet-operator.v2/tune/mx_job_tune_gpu.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: "kubeflow.org/v1alpha2"
+kind: "MXJob"
+metadata:
+  name: "auto-tuning-job"
+spec:
+  jobMode: MXTune
+  mxReplicaSpecs:
+    TunerTracker:
+      replicas: 1
+      restartPolicy: Never
+      template:
+        spec:
+          containers:
+          - name: mxnet
+            image: kingonthestar/auto-tuning:2.0
+            command: ["python3"]
+            args: ["/home/start-job.py"]
+    TunerServer:
+      label: gpu1
+      replicas: 1
+      restartPolicy: Never
+      template:
+        spec:
+          containers:
+          - name: mxnet
+            image: kingonthestar/auto-tuning:2.0
```

Comment:
the image name should be more official , we can use a specific account to store images. And the user can choose what they need from that account